rest/importtest/import_test.go

```diff
@@ -108,6 +109,12 @@ func TestImportFeedWithRecursiveSyncFunction(t *testing.T) {
 	base.RequireWaitForStat(t, rt.GetDatabase().DbStats.SharedBucketImportStats.ImportErrorCount.Value, 2)
 }
 
+func getRawDocWithOnDemandImport(rt *rest.RestTester, docID string) *rest.TestResponse {
+	// trigger on-demand import - since 4.0 _raw will not do this
+	rt.TriggerOnDemandImport(docID)
+	return rt.SendAdminRequest("GET", "/{{.keyspace}}/_raw/"+docID, "")
+}
```

Comment:
I'd be tempted to change the code for 2 reasons:

1. Check for status in this function since it seems like the caller would always have to do the check
2. Unmarshal document for fewer lines of code in the callers, and avoid accidentally unmarshalling into the same output struct without clearing it

```suggestion
// getRawDocWithOnDemandImport does an on demand import and the
func getRawDocWithOnDemandImport(rt *rest.RestTester, docID string) *rest.RawDocResponse {
	// trigger on-demand import - since 4.0 _raw will not do this
	rt.TriggerOnDemandImport(docID)
	resp := rt.SendAdminRequest("GET", "/{{.keyspace}}/_raw/"+docID, "")
	rest.RequireStatus(t, resp, http.StatusOK)
	var output rest.RawDocResponse
	require.NoError(t, base.JSONUnmarshal(resp.BodyBytes(), &output))
	return output
}
```